earth_enterprise/docker/image-definition/scripts/build-gee.sh

```diff
@@ -1,4 +0,0 @@
-#! /bin/bash
```

Comment:
I don't think that's going to work in a Docker build.  The `docker/image-definition/scripts/build-gee.sh` script will be copied to the image, but the `../../../../scripts/build-gee.sh` one won't be.

You'd want to change the `Dockerfile.stage-1.*` scripts to use `build-gee.sh` from the cloned repository, and remove `build-gee.sh` from `image-definition/scripts`.

The same could be done for `build-portable-server.sh`.